src/binary/create-logger.ts

```diff
@@ -101,15 +101,18 @@ export function createLogger(flags: LoggerFlags): Logger {
 }
 
 // Heavily based on https://github.com/winstonjs/logform/blob/master/simple.js
-function simpleFormat() {
+function simpleFormat(stackTrace: boolean) {
   return winston.format((info: winston.Logform.TransformableInfo) => {
-    let stringifiedRest = jsonStringify({
-      ...info,
-      level: undefined,
-      message: undefined,
-      label: undefined,
-    });
-    stringifiedRest = stringifiedRest !== '{}' ? ` ${stringifiedRest}` : '';
+    let stringifiedRest = '';
+    if (stackTrace) {
+      stringifiedRest = jsonStringify({
+        ...info,
```

Comment:
This doesn't seem to do what I want (I need to compile and run).

I expect extra info (like `{msgs}`) to be passed along normally. Just not the stack trace from error messages. Can you manually test this? I think there is a different place that needs adjusting (probably in the error capturing function)

Reply:
Ok, I got this now: 493bf08